Source/FicsItNetworksMisc/Private/ModuleSystem/FINModuleSystemHolo.cpp

```diff
@@ -8,6 +8,17 @@
 AFINModuleSystemHolo::AFINModuleSystemHolo() {
 	PrimaryActorTick.bCanEverTick = true;
 	SetActorTickEnabled(true);
+
+	InformationComponent = CreateDefaultSubobject<UTextRenderComponent>(TEXT("InformationDisplay"));
+	InformationComponent->SetupAttachment(RootComponent);
+	InformationComponent->SetMobility(EComponentMobility::Movable);
+	InformationComponent->SetCollisionEnabled(ECollisionEnabled::NoCollision);
+	
+	if(EnableInformationDisplay) {
+		InformationComponent->SetVisibility(true);
+	}else{
+		InformationComponent->SetVisibility(false);
+	}
```

Comment:
Because `EnableInformationDisplay` is intended to be changed in BP defaults, this code should be moved to OnConstruction or BeginPlay

Reply:
Moved as requested